composer.json

```diff
@@ -29,7 +29,11 @@
     "symfony/console": "^3.4",
     "namshi/cuzzle": "^2.0",
     "rtheunissen/guzzle-log-middleware": "^0.4.1",
-    "mockery/mockery": "^1.0"
+    "mockery/mockery": "^1.0",
+    "symfony/phpunit-bridge": "5.2.x-dev#f2f94fd78379cdcdef09dd5025af791301913968",
```

Comment:
Curious why we're pinning this?

Reply:
Honestly I don't remember. I feel like I would have either a) had a good reason, or b) read it somewhere as recommended for this package. I'm going to go ahead and trust my former self on this one and leave it be :smile: